src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -263,21 +264,21 @@ public boolean sessionHasQuestionsForGiverType(
     }
 
     /**
-     * Gets the recipients of a feedback question.
+     * Gets the recipients of a feedback question including recipient section and team.
      *
      * @param question the feedback question
      * @param instructorGiver can be null for student giver
      * @param studentGiver can be null for instructor giver
      * @param courseRoster if provided, the function can be completed without touching database
-     * @return a map which keys are the identifiers of the recipients and values are the names of the recipients
+     * @return a Map of {@code FeedbackQuestionRecipient} as the value and identifier as the key.
      */
-    public Map<String, String> getRecipientsOfQuestion(
+    public Map<String, FeedbackQuestionRecipient> getRecipientsOfQuestion(
             FeedbackQuestionAttributes question,
             @Nullable InstructorAttributes instructorGiver, @Nullable StudentAttributes studentGiver,
             @Nullable CourseRoster courseRoster) {
         assert instructorGiver != null || studentGiver != null;
 
-        Map<String, String> recipients = new HashMap<>();
+        Map<String, FeedbackQuestionRecipient> recipients = new HashMap();
```

Comment:
```suggestion
        Map<String, FeedbackQuestionRecipient> recipients = new HashMap<>();
```
The one you have right now is raw typing, which in general is not preferred.